scale-tests/configs/kafka-options.json

```diff
@@ -1,5 +1,10 @@
 {
+    "service": {
+        "virtual_network_enabled": true,
+        "virtual_network_name": "dcos"
+    },
     "brokers": {
-        "count": 5
+        "count": 5,
+        "disk_type": "ROOT"
```

Comment:
Is the `ROOT` volume change intentional?

Reply:
Yes, it's to remind me that it needs to be changed to `MOUNT` (once we have the mount volumes).